lib/teaspoon/engine.rb

```diff
@@ -51,6 +51,9 @@ def self.add_precompiled_assets(assets)
 
     def self.inject_instrumentation
       Sprockets::Environment.send(:include, Teaspoon::SprocketsInstrumentation)
+      Sprockets::CachedEnvironment.send(:include, Teaspoon::SprocketsInstrumentation)
+    rescue NameError
```

Comment:
I think it would be better if the rescue was only for Sprockets::CachedEnvironment and not for Sprockets::Environment. We might want to use ```defined? Sprockets::CachedEnvironment```